common/nymcoconut/src/scheme/issuance.rs

```diff
@@ -343,37 +349,39 @@ mod tests {
     #[test]
     fn blind_sign_request_bytes_roundtrip() {
         let mut params = Parameters::new(1).unwrap();
-        let public_attributes = params.n_random_scalars(0);
         let private_attributes = params.n_random_scalars(1);
-        let elgamal_keypair = elgamal::elgamal_keygen(&params);
+        let public_attributes = params.n_random_scalars(0);
 
-        let lambda = prepare_blind_sign(
-            &mut params,
-            &elgamal_keypair,
-            &private_attributes,
-            &public_attributes,
-        )
-        .unwrap();
+        let (_commitments_openings, lambda) =
+            prepare_blind_sign(&mut params, &private_attributes, &public_attributes).unwrap();
 
         let bytes = lambda.to_bytes();
-        println!("{:?}", bytes.len());
         assert_eq!(
             BlindSignRequest::try_from(bytes.as_slice()).unwrap(),
             lambda
         );
 
         let mut params = Parameters::new(4).unwrap();
-        let public_attributes = params.n_random_scalars(2);
         let private_attributes = params.n_random_scalars(2);
-        let lambda = prepare_blind_sign(
-            &mut params,
-            &elgamal_keypair,
-            &private_attributes,
-            &public_attributes,
-        )
-        .unwrap();
+        let public_attributes = params.n_random_scalars(2);
+        let (_commitments_openings, lambda) =
+            prepare_blind_sign(&mut params, &private_attributes, &public_attributes).unwrap();
 
         let bytes = lambda.to_bytes();
+        println!(
```

Comment:
It doesn't really make sense to print anything to `stdout` during tests. If those values are relevant, can we somehow compare them against "good" values or something instead?

Reply:
Removed. Checks are already in place, so this should be OK.